libs/examples/src/index.ts

```diff
@@ -21,7 +21,6 @@
 // tslint:disable: max-file-line-count
 
 import { Type } from '@angular/core';
-
```

Comment:
Can you please undo the import sort here. This file includes far more changes than it would need to.